packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/Snapshot.class.js

```diff
@@ -0,0 +1,30 @@
+export default class NetApp {
+  constructor({ id, name, description, createdAt, type, status }) {
+    Object.assign(this, {
+      id,
+      name,
+      description,
+      createdAt,
+      type,
+      status,
+    });
+  }
+
+  isActive() {
+    return [
+      'available',
+      'manage_starting',
+      'manage_error',
+      'unmanage_starting',
+      'unmanage_error',
+    ].includes(this.status);
+  }
+
+  isInactive() {
+    return ['error'].includes(this.status);
```

Comment:
Maybe add an enum in a constants file?

Reply:
As aim is to only go through class to check status, I'm not sure we should expose an enum, especially it is only used once currently

Reply:
same as above comment